.devcontainer/postAttachCommand.sh

```diff
@@ -1,9 +1,17 @@
-#!/bin/bash -eu
 #
```

Comment:
shebang is removed

Reply:
I think `make license` needs to be changed to avoid this. I'll take a look. Please revert this change manually for now.

Reply:
I'm now working on it

Reply:
opps

Reply:
Please rebase this.
https://github.com/vdaas/vald/pull/2313